### PR TITLE
feat(amf): agw changes to accept amf configs

### PR DESCRIPTION
**Pull request by RahulKalsangra**
Signed-off-by: RahulKalsangra <rahul.kalsangra@wavelabs.ai>

## Summary
This PR deals with the changes as mentioned in the [document ](https://docs.google.com/document/d/1yywmytLNr8p704m7rFcAfHGanG90ZeIo/edit#).
AGW changes for amf configs.
Changed the amf specific parameter names.
Orc8r changes are in #9756.

## Test Plan

UERANSIM
testss
ueransim

### CODE REVIEW

**Review by ulaskozat — APPROVED** [review]
LGTM

**Comment by ulaskozat** [comment]
This can break 5G  until staging orc8r is updated with #9756. You should note the issue as breaking change on backward compatibility.

**Review by ulaskozat — CHANGES_REQUESTED** [review]
where are the pb files for proto? (cc @panyogesh @hcgatewood)

**Comment by ulaskozat** [comment]
@quentinDERORY why is insync checks are not done on this PR? there are updated protos in the PR but pb files are missing, so the check would have failed.

**Comment by hcgatewood** [comment]
> @quentinDERORY why is insync checks are not done on this PR? there are updated protos in the PR but pb files are missing, so the check would have failed.

Hm yeah @quentinDERORY any thoughts on this?

**Comment by quentinDERORY** [comment]
Thanks to @themarwhal, typo in the path filter of the job! Just merged the fix, you should rebase to trigger the workflow

**Comment by RahulKalsangra** [comment]
> where are the pb files for proto? (cc @panyogesh @hcgatewood)

comment addressed and pb files are added

File: lte/gateway/python/scripts/generate_oai_config.py [hunk 1]
```diff
@@ -282,11 +282,14 @@ def _get_default_slice_service_type_config(service_mconfig: object) -> str:
...
-        'mme', 'default_slice_service_type', None,
+        'mme', 'amf_default_slice_service_type', None,
     )
 
     if enable_default_service_slice_type is not None:
-        return enable_default_service_slice_type
+        if isinstance(enable_default_service_slice_type, int):
+            return str(enable_default_service_slice_type)
+        else:
+            return enable_default_service_slice_type
```
Comment (electronjoe) [inline lte/gateway/python/scripts/generate_oai_config.py]: Please review PEP finding `WPS503 Found useless returning `else` statement`

**Comment by panyogesh** [comment]
Some extra commit went in which was not required.  Will be closing the same. This will be replaced by #10392

**Pull request closed without merging**